Swabbr.Api/Controllers/VlogLikeController.cs

```diff
@@ -0,0 +1,155 @@
+﻿using AutoMapper;
+using Microsoft.AspNetCore.Mvc;
+using Swabbr.Api.DataTransferObjects;
+using Swabbr.Api.Extensions;
+using Swabbr.Core.Interfaces.Services;
+using Swabbr.Core.Types;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Swabbr.Api.Controllers
+{
+    /// <summary>
+    ///     Controller for handling requests vlog like operations.
+    /// </summary>
+    [ApiController]
+    [Route("vlog-like")]
+    public sealed class VlogLikeController : ControllerBase
+    {
+        private readonly IVlogLikeService _vlogLikeService;
+        private readonly IMapper _mapper;
+
+        /// <summary>
+        ///     Create new instance.
+        /// </summary>
+        public VlogLikeController(IVlogLikeService vlogLikeService,
+            IMapper mapper)
+        {
+            _vlogLikeService = vlogLikeService ?? throw new ArgumentNullException(nameof(vlogLikeService));
+            _mapper = mapper ?? throw new ArgumentNullException(nameof(mapper));
+        }
+
+        // GET: api/vlog-like/exits/{vlogId}/{userId}
+        /// <summary>
+        ///     Get a vlog.
+        /// </summary>
+        [HttpGet("exists/{vlogId}/{userId}")]
+        public async Task<IActionResult> ExistsAsync([FromRoute] Guid vlogId, [FromRoute] Guid userId)
+        {
+            // Act.
+            var exists = await _vlogLikeService.ExistsAsync(new VlogLikeId
+            {
+                VlogId = vlogId,
+                UserId = userId
+            });
+
+            // Return.
+            return Ok(exists);
+        }
+
+        // GET: api/vlog-like/{vlogId}/{userId}
+        /// <summary>
+        ///     Get a vlog.
```

Comment:
```suggestion
        ///     Get a vlog like.
```

Reply:
Already pushed that because I noticed it too, but you beat me to it.